css/block.css

```diff
@@ -242,6 +242,53 @@ wb-expression[type=rect]:before{ background-image: url(../images/icon/rect.svg);
 wb-expression[type=string]:before{ background-image: url(../images/icon/string.svg); }
 wb-expression[type=geolocation]:before{ background-image: url(../images/icon/geolocation.svg); }
 wb-expression[type=size]:before{ background-image: url(../images/icon/size.svg);}
+
+/* block background colour by category*/
+wb-expression.control, wb-step.control, wb-context.control{ background-color: hsl(0, 93%, 56%); border-color: hsl(0, 93%, 40%); }
```

Comment:
If these class names are not being used for any other elements, the name of the elements can be dropped in these selectors. That is, from `wb-expression.control, wb-step.control, wb-context.control` to `.control`.

Though that might be a bad idea too, since we might reuse these class names for things unrelated to blocks and expressions.